web/modules/custom/solution/src/Controller/SolutionController.php

```diff
@@ -58,6 +59,9 @@ public function createSolutionAccess(RdfInterface $rdf_entity) {
     if ($rdf_entity->bundle() == 'collection' && $this->currentUser()->hasPermission('propose solution rdf entity')) {
       return AccessResult::allowed();
     }
+    if (OgAccess::userAccess($rdf_entity, 'propose solution rdf entity')->isAllowed()) {
+      return AccessResult::allowed();
+    }
```

Comment:
This is a workaround, not the correct solution, this needs a followup ticket and a @todo with a link to the ticket.

This will also start failing soon, the entire `OgAccess` class has been refactored.